content/contributor/content/entities.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Entities
+url: /contributor/content/entities
+weight: 3
+---
+
+Each individual piece of content in the world (such as a scene or a wearable item) is called an _entity_.
+
+Entities are immutable packages of [files]({{< relref "filesystem" >}}) with a unique string identifier, deterministically derived from the contained data, which can be used to discover and download the related files from the content server.
```

Comment:
Not sure about this suggestion, but it might be useful to describe what are these multiple files for:

```suggestion
Entities are immutable packages of [files]({{< relref "filesystem" >}}) with a unique string identifier, deterministically derived from the contained data, which can be used to discover and download the related files from the content server. Entities might require multiple files to be rendered, reproduced, or displayed. For example, a wearable entity might contain a JSON file describing its name and other properties, multiple 3D meshes related to different body types (such as a "male" or "female" avatar body), textures for 3D meshes and 2D representations in the user's inventory. 
```